inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -318,15 +318,15 @@ def test_success_add_gpg_key_method(self, mock_verify_source):
               patch("dispatcher.source.ubuntu_source_manager.add_gpg_key")):    
             command.add(parameters)
 
-    def test_raises_when_space_check_fails(self):
+    def test_raises_when_name_check_fails(self):
```

Comment:
Update the test case `test_raises_when_name_check_fails` to include a test for the new exception message regarding invalid characters in the file name. [important]